### DISCUSSION TITLE
Proposal: DuckStation as a runner.

### DISCUSSION THREAD

**Discussion by mintymoons** [lutris/lutris#5498] [Show and tell]
Greetings, @strycore and Team Lutris.

I propose the addition of DuckStation as a runner for a future version of Lutris.

These are the used files:
**__init__.py (line 34):** [https://github.com/mintymoons/lutris/blob/duckstation-runner/lutris/runners/__init__.py](url)
**duckstation.py (created):** [https://github.com/mintymoons/lutris/blob/duckstation-runner/lutris/runners/duckstation.py](url)

These are some GiFs:
![Install](https://github.com/lutris/lutris/assets/171392014/c3f8ac55-07ed-4391-8e2f-8f063f268d36)
![Settings](https://github.com/lutris/lutris/assets/171392014/135aa395-bc85-4ebb-84be-6e6641873c6c)
![Add a game](https://github.com/lutris/lutris/assets/171392014/f30aace2-d61c-41eb-9b06-07477783d319)
![Runner](https://github.com/lutris/lutris/assets/171392014/23fc5b6c-f743-4b24-8d52-d0aa290fc48f)
![Uninstall](https://github.com/lutris/lutris/assets/171392014/a55cde91-332f-4b2c-b076-e733e7ec8e8f)

Thanks for your attention.

**Comment by danieljohnson2** [comment]
Those links are 404s or me.

Maybe you should submit a PR of this?

**Reply by mintymoons** [reply]
Greetings, @danieljohnson2.

I apologise for this inconvenience. I tried that but when I created a branch, GitHub created a fork.

Here are the files: [files.zip](https://github.com/user-attachments/files/15522307/files.zip)

**Reply by danieljohnson2** [reply]
That's how you do it.

You can't create a branch in the Lutris repo unless you know the secret handshake.

So, fork Lutris, and make a branch if you like, then make your changes, and then submit the PR from there.

**Reply by mintymoons** [reply]
Thank you, very much.

I submitted this PR: https://github.com/lutris/lutris/pull/5499
